server/mentions.go

```diff
@@ -318,13 +320,11 @@ func (p *NotificationsParser) sendUserActivity(userActivity *UserActivity) error
 type UserActivity struct {
 	UserNotification *UserNotification
 	Users            []*model.User
-	TenantID         string
 }
 
-func NewUserActivity(mention *UserNotification, users []*model.User, tenantID string) *UserActivity {
+func NewUserActivity(mention *UserNotification, users []*model.User) *UserActivity {
 	return &UserActivity{
 		UserNotification: mention,
 		Users:            users,
-		TenantID:         tenantID,
```

Comment:
If we manage to support multi-tenant architectures in the future we will need this back so we can retrieve the proper notificationparser from the tenantID here. But right now it's not needed.

Reply:
@fmartingr   Are you sure?  UserActivity is only ever passed to `NotificationsParser.sendUserActivity` which already knows the tenant id.